crates/nu-protocol/src/ast/call.rs

```diff
@@ -130,6 +130,18 @@ impl Call {
         None
     }
 
+    pub fn get_flag_span(&self, flag_name: &str) -> Option<Span> {
+        for name in self.named_iter() {
+            if flag_name == name.0.item {
+                if let Some(val) = &name.2 {
+                    return Some(val.span);
+                }
+            }
+        }
+
+        None
+    }
```

Comment:
While I see the usefulnesss of this little helper and there is nothing inherently wrong with it, it feels a bit odd to add it for just two uses. `nu_engine::CallExt::get_flag<Spanned<T: FromValue>>` is probably what is used in most places.

Reply:
I didn't know that 
Will resolve on next commit 
Thanks